perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -989,7 +989,8 @@ sub get_original_sub : Private {
     my $p = $c->model('DB::Problem')->search({
         category => 'Garden Subscription',
         title => 'Garden Subscription - New',
-        extra => { like => '%property_id,T5:value,I_:'. $c->stash->{property}{id} . '%' }
+        extra => { like => '%property_id,T5:value,I_:'. $c->stash->{property}{id} . '%' },
+        state => { '!=' => 'hidden' },
```

Comment:
Should this be visible_states, or do you sometimes need to fetch unconfirmed?

Reply:
It needs to fetch unconfirmed as the reports are unconfirmed while we wait for the DD to process.